### ISSUE TITLE
A11y: Fix keyboard accessibility in `public/app/plugins/datasource/loki/components/LokiCheatSheet.tsx`

### ISSUE THREAD

**Issue by matyax** [grafana/grafana#59048]
- ⏱️ temporarily enable the eslint rules `jsx-a11y/click-events-have-key-events` and `jsx-a11y/no-static-element-interactions`
  - this can be done by changing them from `"off"` to `"error"` in your `.eslintrc`
- 🔍 open the file to see where the error is (or run `yarn lint` if your editor isn't set up to show eslint errors)
- 🔧 fix the error
  - this will be heavily dependant on context
    - should you be able to interact with this element via the keyboard?
    - is there another element that already handles the same behaviour via the keyboard?
  - in some cases, the best fix is to **use the correct semantic element** for the job. if something behaves like a button but we want it to look different, we can still use the `<button type="button">` element and style it appropriately. this will handle all our keyboard accessibility needs for us. there's even a handy [`clearButtonStyles`](https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Button/Button.tsx#L306) util to undo the basic button styling.
  - if using a `<button>` or other similar interactive element is not appropriate, refer to the documentation [here](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/click-events-have-key-events.md) and [here](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-static-element-interactions.md). this will likely involve setting a `role` and implementing a custom `onKeyPress`
- 🧪 test it! can you interact with this element using the keyboard now?
- 🥳 turn the rules back off and deliver a PR with the fix! 

See https://github.com/grafana/grafana/issues/58924

**Comment by mellieA** [comment]
@matyax are you self-assigning these to work on them later? I noticed this one doesn't have a PR attached.

**Comment by matyax** [comment]
See https://github.com/grafana/grafana/issues/59046#issuecomment-1323320836

**Comment by matyax** [comment]
Fixed in https://github.com/grafana/grafana/pull/59355
